scripts/moneat-agent/moneat_agent/worktree.py

```
@@ -0,0 +1,135 @@
+    import json as _json
+
+    for candidate in (wt_path / ".cursor" / "worktrees.json", repo_root() / ".cursor" / "worktrees.json"):
+        if candidate.exists():
+            data = _json.loads(candidate.read_text())
+            cmds = data.get("setup-worktree-unix") or data.get("setup-worktree", [])
+            if isinstance(cmds, str):
+                cmds = [cmds]
+            env = {**os.environ, "ROOT_WORKTREE_PATH": str(repo_root())}
+            for cmd in cmds:
+                log.system(f"worktree setup: {cmd}")
+                result = subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)